src/css/property-descriptors/background-position.ts

```diff
@@ -11,9 +11,13 @@ export const backgroundPosition: IPropertyListDescriptor<BackgroundPosition> = {
     initialValue: '0% 0%',
     type: PropertyDescriptorParsingType.LIST,
     prefix: false,
-    parse: (_context: Context, tokens: CSSValue[]): BackgroundPosition => {
```

Comment:
This change introduces no improvement and is actually more confusing due to unnamed variables.